src/theory/bv/bitblast/node_bitblaster.h

```diff
@@ -28,12 +28,12 @@ namespace bv {
  *
  * Implements the bare minimum to bit-blast bit-vector atoms/terms.
  */
-class NodeBitblaster : public TBitblaster<Node>
+class NodeBitblaster : public TBitblaster<Node>, protected EnvObj
```

Comment:
I would prefer that `TBitblaster` derives from `EnvObj`, but we discussed this offline and the conclusion is that `TBitblaster` will become obsolete soon, so this is fine.